src/index.ts

```diff
@@ -51,7 +51,7 @@ async function main () {
   const bitcoinMapping: BitcoinDatasource = {}
   const nodeProvider: RSKNodeProvider = {}
   environment.NETWORKS.forEach(network => {
-    dataSourceMapping[network.ID] = new RSKExplorerAPI(network.API_URL, network.CHAIN_ID, axios, network.ID)
+    dataSourceMapping[network.ID] = new BlockscoutAPI(network.API_URL, network.CHAIN_ID, axios, network.ID)
```

Comment:
Before we merge this, let's have both of these lines, but comment out the Blockscout one so the RSK Explorer is the default. I suspect we might use the blockscout one but I want to double check with product.

Reply:
Done!